src/istgt_iscsi.c

```diff
@@ -7180,6 +7203,42 @@ istgt_iscsi_drop_all_conns(CONN_Ptr conn)
 	return (0);
 }
 
+// istgt_iscsi_is_existing_conn_alive returns 1 if
+// there are any existing connections that are alive
+// else return 0
+static int
+istgt_iscsi_is_existing_conn_alive(CONN_Ptr conn)
+{
+	CONN_Ptr existing_conn = NULL;
+	int i;
+
+	MTX_LOCK(&g_conns_mutex);
+	for (i = 0; i <= g_max_connidx; i++) {
+		existing_conn = g_conns[i];
+		if (existing_conn == NULL)
+			continue;
+		if (existing_conn == conn)
+			continue;
+		// If new connection is totaly different from existing connection
+		// then check the status of existing connection if it is active
+		// return true
+		if (strcasecmp(conn->initiator_port, existing_conn->initiator_port) != 0
```

Comment:
what is difference between initiator_port and initiator_name?

Reply:
initiator_name is iSCSI client(initator) unique name whereas
`initiator_port` is initiator_name + initiator_session_id(isid)... I think checking for initiator_port over here is no use(I followed from funcs).

Reply:
updated the check to compare with initiator_address